chopsticks/config.yaml

```diff
@@ -0,0 +1,17 @@
+endpoint:
+    # We fetch relevant storage elements from Spiritnet even tho we are running Peregrine (with sudo).
+    - wss://spiritnet.kilt.io
+db: db/peregrine.sqlite
+runtime-log-level: 5
+port: 50002
+wasm-override: /home/antonio/Developer/kilt-node/target/debug/wbuild/peregrine-runtime/peregrine_runtime.wasm
+
+import-storage:
+    System:
+        Account:
+            - - - 4seWojfEHrk5YKPahdErazQ3CWEHZYi6NV4gKz5AaejWbRPJ
```

Comment:
Is this Alice?

Reply:
Was it? I think it's Alice but with Ed25519.